scripts/zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,576 @@
+    label_only_span = worst_probe["typed_bytes"] - best_probe["typed_bytes"]
+    best_probe_reduction = canonical["typed_bytes"] - best_probe["typed_bytes"]
+    if best_probe["typed_bytes"] <= TWO_PROOF_FRONTIER_TYPED_BYTES:
+        raise RmsnormLabelSensitivityError("label probe unexpectedly beats frontier")
+    if best_probe["typed_bytes"] < COMPACT_SELECTOR_TYPED_BYTES:
+        raise RmsnormLabelSensitivityError("label probe unexpectedly beats compact selector")
+    if best_probe_reduction != 592 or label_only_span != 1_264:
+        raise RmsnormLabelSensitivityError("label sensitivity budget drift")
+    if best_probe_reduction < REQUIRED_REDUCTION_TO_BEAT_FRONTIER_BYTES:
+        pass
+    else:
+        raise RmsnormLabelSensitivityError("label-only reduction reached frontier budget")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)